workflows/autometa.sh

```diff
@@ -231,10 +231,10 @@ then
         set -x
```

Comment:
Would you mind adding a comment of an example header?

```suggestion
        # <insert example ORF header>
        set -x
```